src/js/core.js

```diff
@@ -553,6 +554,27 @@ function MediumEditor(elements, options) {
         if (action === 'image') {
             return this.options.ownerDocument.execCommand('insertImage', false, this.options.contentWindow.getSelection());
         }
+        /* Issue: https://github.com/yabwe/medium-editor/issues/595
+         * If the action is to justify the text */
+        if (justifyAction.exec(action)) {
+            var reg = /<br\/*>/gm,
+                html = Selection.getSelectionHtml.call(this);
+            /* The problem is caused by the <br> that are translated by Chrome in "div" instead of plain text */
+            if (reg.exec(html) !== null) {
+                var range = Selection.getSelectionRange(this.options.ownerDocument);
+                if (range !== null) {
+                    var parentNode = Selection.getSelectedParentElement(range),
+                        temp = html.replace(reg, '');
+                    /* Before the action is applied, alle the <br> are removed. */
```

Comment:
Yes, typo. I coded in line and did not make a function yet because I was not sure about code structure.